install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -147,6 +147,15 @@ containers:
 {{- end }}
 {{- if .Values.global.trustDomain }}
   - --trust-domain={{ .Values.global.trustDomain }}
+{{- end }}
+{{- if $.Values.global.sds.enabled }}
```

Comment:
Also the pattern in this file is to allow override by annotations - so it can be gradually adopted.